docs/backend.md

```diff
@@ -8,6 +8,25 @@ For consistency, we use [Faraday](https://github.com/lostisland/faraday)
 when making HTTP requests. We also wire in notifications so we can
 [log metrics on these requests](../config/initializers/faraday.rb)
 
+```ruby
+conn = Faraday.new do |f|
+  # request_metric is logged specifically as a metric to allow for quicker data aggregation and
+  # historical querying
+  # f.request :instrumentation, name: 'request_metric.faraday'
```

Comment:
Maybe I comment these lines?

Reply:
I wasn't sure how to handle that since only one of them is valid?

Reply:
Could do two separate complete examples?

```rb
# request_metric is logged specifically as a metric to allow for quicker data aggregation and
# historical querying
conn = Faraday.new do |f|
  f.request :instrumentation, name: 'request_metric.faraday'
end

# request_log is logged, but only to the log file, typically for requests where we are
# less interested in aggregation and generally only need to view the attributes of a specific
# request
conn = Faraday.new do |f|
  f.request :instrumentation, name: 'request_log.faraday'
end
```